src/styles/tears-of-the-wild.css

```diff
@@ -0,0 +1,125 @@
+pre code.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 1em
+}
+code.hljs {
+    padding:3px 5px
+}
```

Comment:
These lines can be removed, our build system adds these to the CSS when it builds the projects.